app/javascript/angular/tests/_draw_session.test.js

```diff
@@ -1,17 +1,6 @@
 import test from "blue-tape";
 import { mock } from "./helpers";
-import { drawSession } from "../code/services/_draw_session.js.erb";
-
-test("drawMobileSession draws a session when session is loaded and sensor is selected", t => {
```

Comment:
When changing the filename some tests stop being run. In fact `yarn test` == `$ NODE_ENV=test blue-tape -r @babel/register app/javascript/angular/tests/**/*.test.js`. The suffix `.erb` was not matching that dir glob.

This test fails cause the module does `new google....` and being `google` a global it fails. Since it's easier to delete and just a unit test I'm removing it.